org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -897,13 +929,10 @@ class CodeAction {
     new() {
     }
     
-    new(@NonNull String title, String kind, List<Diagnostic> diagnostics, WorkspaceEdit edit, Command command) {
+    new(@NonNull String title) {
```

Comment:
Won't it be a breaking change? Or we don't care?

Reply:
The class is new in 0.5.0. I aligned it with the rest of our implementation, where we specify constructors only when there are only few properties.